brightener.cpp

```diff
@@ -1,4 +1,6 @@
 #include "brightener.h"
+constexpr int maz_size = 255;
```

Comment:
Good approach to name the constants.
Can we improve the names further? What does `maz_size` mean? Size of what?
In this, 255 would be the max brightness possible...